### PR TITLE
Update to Angular 7

### PR DESCRIPTION
**Pull request by GregOnNet**
Hi,

here is what I did to update `ngx-drag-to-select` to use Angular 7.

### ng update

`ng update @angular/cli @angular/core`

### Manually Updated Dependencies

- @angular/material
- codelyzer
- @nguniversal

### Executed scripts to check if everything still works

- ✅`yarn start`
- ✅`yarn build`
- ✅`yarn build:lib`
- ✅ `yarn build:ssr`
- ✅ `yarn serve:ssr`
- ✅`yarn test`
- ✅`yarn e2e`

Please let me know if I have missed something.
Hopefully, this PR helps to resolve Issue #57.

### CODE REVIEW

**Comment by GregOnNet** [comment]
Sorry,

I am not able to reproduce the error shown in the build log since I cannot execute `yarn e2e:ci` without the record key.

```
$ cypress run --record
You passed the --record flag but did not provide us your Record Key.

You can pass us your Record Key like this:

  cypress run --record --key <record_key>

You can also set the key as an environment variable with the name CYPRESS_RECORD_KEY.

https://on.cypress.io/how-do-i-record-runs
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Locally, `yarn e2e` passes all tests. 🤔

File: package.json [hunk 1]
```diff
@@ -51,29 +51,29 @@
...
     "core-js": "^2.5.7",
-    "rxjs": "^6.3.2",
+    "rxjs": "^6.3.3",
     "web-animations-js": "^2.3.1",
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.8.3",
+    "@angular-devkit/build-angular": "~0.10.0",
     "@angular-devkit/build-ng-packagr": "^0.8.3",
```
Comment (d3lm) [inline package.json]: What about this package? There's `0.10.3` available on npm. Maybe we should also update this one.

File: src/polyfills.ts [hunk 2]
```diff
@@ -42,7 +42,6 @@
 
 /** Evergreen browsers require these. **/
 // Used for reflect-metadata in JIT. If you use AOT (and only Angular decorators), you can remove.
-import 'core-js/es7/reflect';
```
Comment (d3lm) [inline src/polyfills.ts]: Why is this removed?

**Comment by d3lm** [comment]
Regarding the e2e tests, it's prolly that they need to be restarted. Every once in a while they fail on CI tho they work locally.

File: src/polyfills.ts (hunk 2 above)
Comment (timdeschryver) [reply src/polyfills.ts]: This is done automatically by `ng update`. 
It's because it should only be used in dev mode, but people weren't removing it for prod builds.

File: package.json (hunk 1 above)
Comment (GregOnNet) [reply package.json]: @d3lm @markgoho I am really sorry. I did not saw the ongoing conversation on twitter between both of you.

@d3lm Pick whatever PR you like. It did not take me long to do this.
Nevertheless, I will update the dependency you have mentioned.

Sorry for confusing everyone. 🙏

Comment (GregOnNet) [reply package.json]: Updated ng-packagr

File: src/polyfills.ts (hunk 2 above)
Comment (d3lm) [reply src/polyfills.ts]: Uh good to know!

**Comment by d3lm** [comment]
Thanks again @GregOnNet for your PR. Would it be ok if I proceed with @markgoho PRs because he was first. Unfortunately I cannot merge both but I also don't want to upset anyone. I hope this makes sense and sorry for the work you invested in this. But there will be another chance for contribution.

**Comment by d3lm** [comment]
But that said, your PR looks polished and I especially liked the way you structured your PR message as well as how you approached the update. Well done!

**Comment by GregOnNet** [comment]
Sure, it is most important to that the update is made somehow. :)
Thank you for your feedback and thank you for providing such a nice library. 👍
